packages/components/nodes/memory/MementoMemory/MementoMemory.ts

```diff
@@ -0,0 +1,77 @@
+import { SystemChatMessage } from 'langchain/schema'
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { MomentoChatMessageHistory } from 'langchain/stores/message/momento'
+import { ICommonObject } from '../../../src'
+import { CacheClient } from '@gomomento/sdk'
+
+class MomentoMemory implements INode {
+    label: string
+    name: string
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Momento Memory'
+        this.name = 'MomentoMemory'
+        this.type = 'MomentoMemory'
+        this.icon = 'memory.svg'
```

Comment:
would love to use their official logo - https://github.com/momentohq/client-sdk-javascript